network-api/networkapi/highlights/factory.py

```diff
@@ -0,0 +1,33 @@
+import factory
+from factory.django import DjangoModelFactory
+
+from networkapi.utility.utc import UTC
+from networkapi.highlights.models import Highlight
+
+utc = UTC()
+
+
+class HighlightFactory(DjangoModelFactory):
+    title = factory.Faker('sentence', nb_words=4)
+    description = factory.Faker('paragraphs', nb=2)
+    link_label = factory.Faker('words', nb=3)
+    link_url = factory.Faker('uri')
+    image = factory.LazyAttribute(lambda a: '{}image.png'.format(a.link_url))
+    footer = factory.Faker('sentence', nb_words=5)
+    publish_after = factory.Faker('past_datetime', start_date='-30d', tzinfo=utc)
+    expires = None
+    order = 0
```

Comment:
why do we force `0` always?

Reply:
I think that's the model default - we could probably safely remove it from the Factory.